### PR TITLE
Fix nodeId regression

### PR DESCRIPTION
**Pull request by luiscantero**
* Fix regression caused by [Better NodeId/ExpandedNodeId built in types handling](https://github.com/OPCFoundation/UA-.NETStandard/pull/2230)
* Small SQ and cosmetic improvements

### CODE REVIEW

File: src/PluginNodes/UserDefinedPluginNodes.cs [hunk 1]
```diff
@@ -123,7 +123,11 @@ private IEnumerable<NodeWithIntervals> AddNodes(FolderState folder, ConfigFolder
 
             CreateBaseVariable(userNodesFolder, node);
 
-            yield return PluginNodesHelper.GetNodeWithIntervals((NodeId)node.NodeId, _plcNodeManager);
+            var nodeId = node.NodeId is string
```
Comment (mregen) [inline src/PluginNodes/UserDefinedPluginNodes.cs:126]: can you use isString from above?

**Review by mregen — APPROVED** [review]

**Review by koepalex — APPROVED** [review]

**Review by hansgschossmann — APPROVED** [review]

**Review by cristipogacean — APPROVED** [review]

File: src/PluginNodes/UserDefinedPluginNodes.cs (hunk 1 above)
Comment (luiscantero) [reply src/PluginNodes/UserDefinedPluginNodes.cs]: Will fix, txs

File: version.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "$schema": "https://raw.githubusercontent.com/AArnott/Nerdbank.GitVersioning/master/src/NerdBank.GitVersioning/version.schema.json",
-  "version": "2.9.9",
+  "version": "2.9.10",
```
Comment (karok2m) [inline version.json:3]: Can you please integrate `2.9.10` in project-apollo repo once it is released, similar to https://github.com/Azure/project-apollo/pull/1117.

File: src/DeterministicAlarms/Model/SimConditionStatesEnum.cs [hunk 3]
```diff
@@ -41,22 +41,22 @@ public enum SimConditionStatesEnum
     Shelved = 0x20,
 
     ///// <summary>
-    ///// The condition has exceeed the high-high limit.
+    ///// The condition has exceed the high-high limit.
```
Comment (karok2m) [inline src/DeterministicAlarms/Model/SimConditionStatesEnum.cs:44]: It should be `has exceeded`.

File: src/AlarmCondition/UnderlyingSystem/UnderlyingSystemAlarmStates.cs [hunk 4]
```diff
@@ -74,22 +74,22 @@ public enum UnderlyingSystemAlarmStates
         Shelved = 0x20,
 
         /// <summary>
-        /// The condition has exceeed the high-high limit.
+        /// The condition has exceed the high-high limit.
```
[the remark this answers is not in the document]
Comment (luiscantero) [reply src/AlarmCondition/UnderlyingSystem/UnderlyingSystemAlarmStates.cs:77]: Txs

**Pull request merged**